overrides/defaultconfigs/sophisticatedbackpacks-server.toml

```diff
@@ -390,29 +390,29 @@
 	[server.entityBackpackAdditions]
 		#Chance of an entity spawning with Backpack
 		#Range: 0.0 ~ 1.0
-		chance = 0.0
+		chance = 0.00
 		#Turns on/off addition of loot into backpacks
-		addLoot = true
+		addLoot = false
 		#Turns on/off buffing the entity that wears backpack with potion effects. These are scaled based on how much loot is added.
-		buffWithPotionEffects = true
+		buffWithPotionEffects = false
 		#Turns on/off buffing the entity that wears backpack with additional health. Health is scaled based on backpack tier the mob wears.
-		buffHealth = true
+		buffHealth = false
 		#Turns on/off equiping the entity that wears backpack with armor. What armor material and how enchanted is scaled based on backpack tier the mob wears.
-		equipWithArmor = true
+		equipWithArmor = false
 		#Map of entities that can spawn with backpack and related loot tables (if adding a loot is enabled) in format of "EntityRegistryName|LootTableName"
-		entityLootTableList = ["minecraft:creeper|minecraft:chests/desert_pyramid", "minecraft:drowned|minecraft:chests/shipwreck_treasure", "minecraft:enderman|minecraft:chests/end_city_treasure", "minecraft:evoker|minecraft:chests/woodland_mansion", "minecraft:husk|minecraft:chests/desert_pyramid", "minecraft:piglin|minecraft:chests/bastion_bridge", "minecraft:piglin_brute|minecraft:chests/bastion_treasure", "minecraft:pillager|minecraft:chests/pillager_outpost", "minecraft:skeleton|minecraft:chests/simple_dungeon", "minecraft:stray|minecraft:chests/igloo_chest", "minecraft:vex|minecraft:chests/woodland_mansion", "minecraft:vindicator|minecraft:chests/woodland_mansion", "minecraft:witch|minecraft:chests/buried_treasure", "minecraft:wither_skeleton|minecraft:chests/nether_bridge", "minecraft:zombie|minecraft:chests/simple_dungeon", "minecraft:zombie_villager|minecraft:chests/village/village_armorer", "minecraft:zombified_piglin|minecraft:chests/bastion_other"]
+		entityLootTableList = ["minecraft:creeper|minecraft:chests/desert_pyramid"]
 		#List of music discs that are not supposed to be played by entities
```

Comment:
if the idea is to disabled mobs with backpacks It looks like maybe having this array be empty should do it

Reply:
agreed with this

Reply:
also it doesnt make much sense that needing an extra 0 would do anything. unless Java or toml does things strangely 0 should == 0.0 == 0.00. 

make sure that when you're testing in game it is actually using the edited script!

Reply:
Changed code --> empty array